src/Components/Inquiry/config.ts

```diff
@@ -26,6 +28,26 @@ interface UseEngine {
 export const useEngine = ({ context, onDone }: UseEngine) => {
   const visited = useRef(new Visited("inquiry"))
 
+  const conditions = {
+    askSpecialist: () => {
+      return !!context.current?.askSpecialist
+    },
+    hasBasicInfo: () => {
+      return (
+        !!context.current?.profession &&
+        !!context.current?.location?.city &&
+        !!context.current?.otherRelevantPositions &&
+        !!context.current?.shareFollows
+      )
+    },
+    isLoggedOut: () => {
+      return !context.current?.isLoggedIn
+    },
+    hasArtistsInCollection: () => {
+      return (context.current?.userInterestsConnection?.totalCount ?? 0) > 0
```

Comment:
My understanding of this feature is that users can add either artworks or artists to their My Collection, and that we want to prompt them to add artists to their My Collection if it is empty, so I expected this check to involve My Collection.

That's why I'm surprised to see you checking user interests here instead, but I have to admit that I don't know what makes up user interests. Are they the same thing?

Reply:
They are indeed the same thing.